modules/bazel_lib/3.0.0-beta.0/patches/go_dev_dep.patch

```diff
@@ -0,0 +1,35 @@
+diff --git a/MODULE.bazel b/MODULE.bazel
+index e63fa5b..9d78a88 100644
+--- a/MODULE.bazel
++++ b/MODULE.bazel
+@@ -50,19 +50,19 @@ use_repo(host, "bazel_lib_host")
```

Comment:
I guess the line numbers here is mismatching the content, but somehow the `patch` tool tried to be smart and failed.
Can you try to re-generate the patch with git?